src/modules/dashboard/sagas/motions/createEditDomainMotion.ts

```diff
@@ -79,13 +79,13 @@ function* createEditDomainMotion({
     txChannel = yield call(getTxChannel, metaId);
 
     // setup batch ids and channels
-    const batchKey = 'createDomainMotion';
+    const batchKey = 'createMotion';
 
     const {
-      createDomainMotion,
+      createMotion,
```

Comment:
I mean... the tx channel name didn't necessarily need renaming, just the `methodName` we pass to the transaction.

But with all that being said, I very much appreciate the extra effort :1st_place_medal: